src/VPolygon.jl

```diff
@@ -585,10 +630,10 @@ function minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
     mP = length(vlistP)
     mQ = length(vlistQ)
     i = 1
-    σP = σ(N[1, 0], P)
-    σQ = σ(N[1, 0], Q)
-    k = findfirst(==(σP), vlistP)
-    j = findfirst(==(σQ), vlistQ)
+    σP = _binary_support_vector(N[1, 0], P)
+    σQ = _binary_support_vector(N[1, 0], Q)
+    k = findfirst(==(vlistP(σP), vlistP))
```

Comment:
Should it not be:
```suggestion
    k = findfirst(==(vlistP[σP], vlistP))
```
Better rename `σP` and `σQ` to make clear that those are indices. Actually, if I understand correctly, `σP == k`, so no need for this line at all (I might be wrong, though).